README.md

```diff
@@ -75,9 +75,12 @@ The bot will send you a private message (make sure your Discord settings allow D
 
 If you want to view command usage or see the available options, type `.au` or `.au help` in your Discord channel.
 
+## Galactus 
+
```

Comment:
Maybe add a link to Galactus somewhere in here? https://github.com/automuteus/galactus